components/brave_ads/core/internal/targeting/contextual/text_classification/resource/text_classification_resource.cc

```diff
@@ -52,33 +57,33 @@ void TextClassificationResource::MaybeLoadOrReset() {
 void TextClassificationResource::Load() {
   did_load_ = true;
 
-  LoadAndParseResource(kTextClassificationResourceId,
-                       kTextClassificationResourceVersion.Get(),
-                       base::BindOnce(&TextClassificationResource::LoadCallback,
-                                      weak_factory_.GetWeakPtr()));
+  LoadFlatBuffersResource(
+      kFlatBuffersTextClassificationResourceId,
+      kFlatBuffersTextClassificationResourceVersion.Get(),
```

Comment:
@aseren is this related to the versioning you removed?

Reply:
No, this is the resource version from resource.json file